spring-cloud-dataflow-core/src/test/java/org/springframework/cloud/dataflow/core/StreamDefinitionToDslConverterTests.java

```diff
@@ -41,6 +41,8 @@ public void quotesInParams() {
 	}
 
 	@Test
+	@Ignore
+	// This DSL isn't a valid
```

Comment:
Based on what is this considered invalid? It is well-formed and parsed correctly by the DSL?

Reply:
for example a `http --port=9700 | filter --expression="payload.matches('hello world') or payload.matches('big world')" | file` is a valid DSL  and the double quotes are needed. 
Written as java string (as in the tests) this dsl will have the double quotes escaped: `http --port=9700 | filter --expression=\"payload.matches('hello world') or payload.matches('big world')\" | file`

Reply:
I agree this is a valid one. When I had it copied into shell, the `\"` got escaped and that led to error which made me think we don't support it.